nox/__main__.py

```diff
@@ -22,9 +22,9 @@
 import sys
 
 try:
-    import importlib.metadata as metadata
-except ImportError:
-    import importlib_metadata as metadata
+    import importlib.metadata as metadata  # type: ignore
+except ImportError:  # pragma: no cover
```

Comment:
Out of curiosity, when would this branch get hit?

Reply:
On Python 3.7 or earlier.